packages/terra-worklist-data-grid/src/WorklistDataGrid.module.scss

```diff
@@ -8,18 +8,24 @@
     outline-offset: var(--terra-worklist-data-grid-cell-focus-outline-offset, -2px);
   }
 
+  .worklist-data-grid-container { 
+    height: 100%;
+    overflow-x: scroll;
+    width: 100%;
+  }
+
   .worklist-data-grid {
     // Style for the grid itself
     border: var(--terra-worklist-data-grid-border, 1px solid #dedfe0);
     border-collapse: collapse;
     table-layout: fixed;
-    width: 100%;
+    width: 1px;
```

Comment:
Is this value correct?

Reply:
Yes, this value will allow the table to expand to the width of its contents.  Some value must be provided to trigger the proper layout logic.